pkg/controller/migmigration/task.go

```diff
@@ -41,6 +41,9 @@ const (
 	EnsureFinalRestore            = "EnsureFinalRestore"
 	FinalRestoreCreated           = "FinalRestoreCreated"
 	FinalRestoreFailed            = "FinalRestoreFailed"
+	ValidateResourcesMigrated     = "ValidateResourcesMigrated"
+	ValidationInProgress          = "ValidationInProgress"
```

Comment:
s/Validation/Verification/g

A separate `InProgress` phase deviates from other phases that can also be long running.  Also, I believe that the `ValidateResourcesMigrated` phase checks the static resources like DCs.  Suggest something like:

VerifyPodOwners
VerifyPods

.. or something :)